packages/tools/src/store/ToolGroupManager/ToolGroup.ts

```diff
@@ -326,8 +326,12 @@ export default class ToolGroup implements IToolGroup {
     const bindingsToUse = [...prevBindings, ...newBindings].reduce(
       (unique, binding) => {
         if (
-          !unique.some((obj) => obj.mouseButton === binding.mouseButton) &&
-          binding.mouseButton !== undefined
+          (!unique.some((obj) => obj.mouseButton === binding.mouseButton) &&
+            binding.mouseButton !== undefined) ||
+          (!unique.some(
+            (obj) => obj.numTouchPoints === binding.numTouchPoints
+          ) &&
+            binding.numTouchPoints !== undefined)
```

Comment:
This has become very cumbersome, can we break it into two checks? `hasMouseBinding` and `hasTouchBinding` (if I got it correctly), or similar naming

Reply:
Can do